docs/source/topics/assembly_accessing-the-openshift-cluster.adoc

```diff
@@ -1,7 +1,7 @@
 [id="accessing-the-openshift-cluster_{context}"]
-= Accessing the OpenShift cluster
+= Accessing the {openshift} cluster
```

Comment:
Just below, it is `Access the {ocp} cluster`, and here it is an `{openshift} cluster`

Reply:
Correct. I tried to keep the titles short by only referring to OpenShift, but using the complete name (OpenShift Container Platform) in the text itself.